test.js

```diff
@@ -13,7 +13,7 @@ function MyPersistence () {
 
   // copied from Memory
   this._retained = []
-  this._subscriptions = []
+  this._subscriptions = new Map()
```

Comment:
Each persistence will choose its own way of storing subscriptions. aedes-persistence-redis, for example, stores it in a Redis hash key.